src/main/resources/application.properties

```diff
@@ -11,4 +11,10 @@ logging.level.com.chessmates=DEBUG
 # Properties used within our application.
 chessmates.lichess.api.game.pageSize=100
 chessmates.lichess.api.user.pageSize=50
-chessmates.lichess.api.dataupdate=true
\ No newline at end of file
+chessmates.lichess.api.dataupdate=true
+
+chessmates.rds.dbName = chessmates
+chessmates.rds.userName = chessmates
+chessmates.rds.password = ${CHESSMATES_RDS_PW}
+chessmates.rds.hostName = chessmates-rds.chrwglr5bu8k.eu-west-1.rds.amazonaws.com
```

Comment:
Can we somehow avoid storing that in the app properties? It feels like this should be a secret as well.